ui/src/App.tsx

```diff
@@ -1,45 +1,28 @@
 import { createContext, useContext } from 'react';
 import { MOCK_REPORT } from './mocks/report.mock';
 import { Report } from '@app/api/report';
-// import { TabbedLayout } from './TabbedLayout';
-import {Grid, GridItem, PageSection, PageSectionVariants, TextContent, Text} from "@patternfly/react-core";
+import { Grid, GridItem, PageSection, PageSectionVariants } from '@patternfly/react-core';
 import { SummaryCard } from './components/SummaryCard';
-import { ChartCard } from './components/ChartCard';
-import { Dependencies } from './components/Dependencies';
-
+import { DependenciesTable } from './components/DependenciesTable';
 
 const data = (window as any)['report'] || MOCK_REPORT;
```

Comment:
@carlosthe19916 would you mind (doesn;t have to be in this PR) to update this to use environment variables and only use MOCK_DATA in dev mode? Then it won't get compiled into the main.js bundle